### DISCUSSION TITLE
Does Wiki.js support Prometheus integration for monitoring and metrics?

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [requarks/wiki#6439] [Help / Questions]
Hello,

I hope you're well and everything is good with you.

Does Wiki.js support Prometheus integration for monitoring and metrics?

Thank you very much.
Best regards,

**Comment by BriianPowell** [comment]
Bumping this as I would also like to know, using the Prometheus operator and wondering if it's possible to get metrics using a ServiceMonitor!

**Comment by sedatsencan** [comment]
Guys, were you able to monitor the wiki.js somehow? :)
